sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -224,7 +224,7 @@ class BlobAPITest extends APISpec {
         bc.download(null)
 
         then:
-        thrown(StorageException)
+        thrown(NullPointerException)
```

Comment:
Is there a better exception that describes that the parameter value is null?

Reply:
This is the most correct exception to be thrown in Java for a parameter that is meant to be non-null and is null.